e2e_playwright/shared/app_utils.py

```diff
@@ -740,3 +740,41 @@ def expect_font(page: Page, font_family: str, timeout: int = 20000) -> None:
     }
     """
     page.wait_for_function(check_script, arg=font_family, timeout=timeout)
+
+
+def is_element_in_bounding_box_of(
```

Comment:
nit: `is_element_in_bounding_box_of` -> `is_child_bounding_box_inside_parents`

and I would probably swap the two arguments, child first and then the parent because the child element is mentioned in the function name first, too

Reply:
Updated 👍